topic-operator/src/test/java/io/strimzi/operator/topic/MockKafka.java

```diff
@@ -60,7 +60,8 @@ public MockKafka setTopicMetadataResponse(Function<TopicName, Future<TopicMetada
         return this;
     }
 
-    public MockKafka setTopicMetadataResponses(Function<TopicName, Future<TopicMetadata>>... topicMetadataRespose) {
+    @SafeVarargs
```

Comment:
Why do we need this?

Reply:
To suppress a warning when using varargs of a parameterised type.
https://www.baeldung.com/java-safevarargs